spec/src/main/java/io/a2a/spec/InvalidAgentResponseError.java

```diff
@@ -44,7 +44,7 @@
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class InvalidAgentResponseError extends JSONRPCError {
 
-    public final static Integer DEFAULT_CODE = -32006;
+    public final static int DEFAULT_CODE = -32006;
```

Comment:
can we get the code from the `.proto` file or somewhere else?

Reply:
If the JSON-RPC error codes have leaked into the domain classes, I don't see why it couldn't also provide the REST mapping?

Reply:
That's minor. I'm just pointing out that the JSON-RPC codes have leaked into the "domain" classes (in `spec`) and are not in the transport layer for JSON-RPC.
That's fine.
My point was that if the InvalidAgentResponseError si the domain representation of this error *and* contains the error code for the JSON-RPC, it could also contains the status code for REST.